### PR TITLE
add aek/too checker function to khavee

### PR DESCRIPTION
**Pull request by HRNPH**
### What does this changes
Add aek/too word checking function into khavee

### What was wrong

there's no aek/too checking functionality in khavee yet

### How this fixes it

Add aek/too word checking function into khavee

Fixes #...

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](https://github.com/PyThaiNLP/pythainlp/blob/dev/CONTRIBUTING.md) to this repository.

- [ ] Passed code styles and structures
- [ ] Passed code linting checks and unit test

### CODE REVIEW

File: pythainlp/khavee/core.py [hunk 1]
```diff
@@ -379,3 +379,38 @@ def check_klon(self, text: str,k_type: int=8) -> Union[List[str], str]:
             
         else:
             return 'Something went wrong Make sure you enter it in correct form.'
+
+    def check_aek_too(self, text: str or List[str]) -> str or bool or List[str or bool]:
```
Comment (wannaphong) [inline pythainlp/khavee/core.py]: - `text: str or List[str]` should be text `Union[List[str], str]`
- `str or bool or List[str or bool]` should be `Union[List[bool], List[str], bool, str]`

File: pythainlp/khavee/core.py [hunk 2]
```diff
@@ -379,3 +379,38 @@ def check_klon(self, text: str,k_type: int=8) -> Union[List[str], str]:
...
         else:
             return 'Something went wrong Make sure you enter it in correct form.'
+
+    def check_aek_too(self, text: str or List[str]) -> str or bool or List[str or bool]:
+        """
+        Check if the word is aek or too or False(not both)
+
+        :param str or list[str] text: Thai word or list of Thai words
+        :return: the check if the word is aek or too or False(not both) or list of the check if input is list
+        :rtype: str or bool or list[str or bool]
```
Comment (wannaphong) [inline pythainlp/khavee/core.py]: `str or bool or list[str or bool]` should be `Union[List[bool], List[str], bool, str]`

File: tests/test_khavee.py [hunk 3]
```diff
@@ -25,3 +25,9 @@ def test_check_klon(self):
             kv.check_klon('''ฉันชื่อหมูกรอบ ฉันชอบกินไก่ แล้วก็วิ่งไล่ หมาชื่อนํ้าทอง ลคคนเก่ง เอ๋งเอ๋งเสียงหมา มีคนจับจอง เขาชื่อน้องเธียร''',k_type=4),
             ["Cant find rhyme between paragraphs ('หมา', 'จอง')in paragraph 2", "Cant find rhyme between paragraphs ('หมา', 'ทอง')in paragraph 2"]
         )
+
+    def test_check_aek_too(self):
+        self.assertEqual(kv.check_aek_too('ไกด์'), False)
+        self.assertEqual(kv.check_aek_too('ไก่'), 'aek')
+        self.assertEqual(kv.check_aek_too('ไก้'), 'too')
+        self.assert_(kv.check_aek_too(['หนม', 'หน่ม', 'หน้ม']), [False, 'aek', 'too'])
```
Comment (wannaphong) [inline tests/test_khavee.py:33]: add blank newline after this line

File: pythainlp/khavee/core.py [hunk 4]
```diff
@@ -379,3 +379,38 @@ def check_klon(self, text: str,k_type: int=8) -> Union[List[str], str]:
             
         else:
             return 'Something went wrong Make sure you enter it in correct form.'
+
+    def check_aek_too(self, text: str or List[str]) -> str or bool or List[str or bool]:
+        """
+        Check if the word is aek or too or False(not both)
```
Comment (wannaphong) [inline pythainlp/khavee/core.py]: `คำเอกโท` in English?

**Review by wannaphong — CHANGES_REQUESTED** [review]
Thank you for this pull request! I have comments that you can fix.

File: pythainlp/khavee/core.py (hunk 4 above)
Comment (HRNPH) [reply pythainlp/khavee/core.py]: thai tonal word checker?
https://dict.longdo.com/search/-%E0%B9%80%E0%B8%AD%E0%B8%81%E0%B9%82%E0%B8%97%E0%B8%A9-

File: pythainlp/khavee/core.py (hunk 1 above)
Comment (HRNPH) [reply pythainlp/khavee/core.py]: changed👍

File: pythainlp/khavee/core.py (hunk 2 above)
Comment (HRNPH) [reply pythainlp/khavee/core.py]: changed👍

File: tests/test_khavee.py (hunk 3 above)
Comment (HRNPH) [reply tests/test_khavee.py:33]: added 👍

**Comment by cstorm125** [comment]
wow

**Review by wannaphong — APPROVED** [review]
💯

**Pull request merged**